src/libraries/System.Runtime/tests/System/Diagnostics/StackTraceHiddenAttributeTests.cs

```diff
@@ -0,0 +1,17 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Diagnostics;
+using Xunit;
+
+namespace System.Tests
+{
+    public static class StackTraceHiddenAttributeTests
```

Comment:
Add a test to validate that `NoOptimization | NoInlining` method with this attribute is actually hidden in the stacktrace?